ReactWindows/ReactNative.Shared/Bridge/ReactInstance.cs

```diff
@@ -113,26 +114,21 @@ public void Initialize()
                     }
 
                     _bundleLoader.LoadScript(_bridge);
+
+                    // Now that the bridge is initialized, queue up all actions which were waiting for bridge
+                    foreach (var action in _onInitBridgeActions)
+                    {
+                        QueueConfiguration.JavaScriptQueue.RunAsync(action);
```

Comment:
At this point, we are on the JavaScriptQueue and as such can directly execute the action, but since this function is probably being awaited in the app launch/resume sequence, it can impact app launch times, so instead requeuing the actions.

This has the effect that the javascript actions might not execute in the exact same order as with earlier implementation, but since the unit tests seems to pass so I am assuming that the exact order of execution is not a strict requirement.